### PR TITLE
[BEAM-13979] [Website] Update Community/Contact us page

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-13979]
- Mailing list, what are they, and how they work (update text, add links)
- Useful Tips for Sending Emails (update text, remove tip 3)
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @AydarZaynutdinov

**Comment by AydarZaynutdinov** [comment]
retest this please

**Comment by AydarZaynutdinov** [comment]
retest this please

File: website/www/site/content/en/community/contact-us.md [hunk 1]
```diff
@@ -23,22 +23,22 @@ limitations under the License.
 
 # Contact us!
 
-The Apache Beam community is friendly and welcoming. We are glad to help with any
-question, suggestion or idea you have. Contact us in the following channels:
+The Apache Beam community is friendly and welcoming. We are glad to help with any
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/community/contact-us.md:26]:
```suggestion
The Apache Beam community is friendly and welcoming. We are glad to help with any
```

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/community/contact-us.md (hunk 1 above)
Comment (bullet03) [reply website/www/site/content/en/community/contact-us.md]: done

**Comment by dariabezkorovaina** [comment]
R: @pcoet

File: website/www/site/content/en/community/contact-us.md [hunk 2]
```diff
@@ -24,21 +24,21 @@ limitations under the License.
...
 
 ### Subscribe and Unsubscribe:
 
 Prior to sending emails to these lists, you need to subscribe. To subscribe, send a blank email to [user-subscribe@beam.apache.org](mailto:user-subscribe@beam.apache.org) or [dev-subscribe@beam.apache.org](mailto:dev-subscribe@beam.apache.org) depending on the list you want to write to.
 
-To unsubscribe, send a blank email to [user-unsubscribe@beam.apache.org](mailto:user-unsubscribe@beam.apache.org) or [dev-unsubscribe@beam.apache.org](mailto:dev-unsubscribe@beam.apache.org) depending on the list you want to unsubscribe.
+Once you subscribe, you’ll receive all emails going to this mailing list. If you no longer wish to receive emails from these lists, you need to send another blank email to either  [user-unsubscribe@beam.apache.org](mailto:user-subscribe@beam.apache.org) or [dev-unsubscribe@beam.apache.org](mailto:dev-subscribe@beam.apache.org).
```
Comment (pcoet) [inline website/www/site/content/en/community/contact-us.md:41]: Nit: "Once" -> "After"

File: website/www/site/data/en/contact_us.yaml [hunk 3]
```diff
@@ -76,19 +76,30 @@
...
+      actions: [
+        {
+          action: "Join the",
+        },
+        {
+          action: "#beam channel",
+          link: "https://app.slack.com/client/T4S1WH2J3/C9H0YNP3P",
+        },
+        {
+          action: " and browse other available channels. Don’t ask Beam questions in #general, we don’t monitor it."
```
Comment (pcoet) [inline website/www/site/data/en/contact_us.yaml:100]: "#general, we don't" -> "#general. We don't"

**Review by pcoet — APPROVED** [review]
One nit (not blocking)

File: website/www/site/data/en/contact_us.yaml (hunk 3 above)
Comment (bullet03) [reply website/www/site/data/en/contact_us.yaml]: changed

File: website/www/site/content/en/community/contact-us.md (hunk 2 above)
Comment (bullet03) [reply website/www/site/content/en/community/contact-us.md]: changed

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

**Pull request merged**